miscellaneous/go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/ElrondNetwork/elrond-tools-go/miscellaneous
```

Comment:
let's rename this to something more meaningful. `miscellaneous` is way to general. Something like `tokensRemoverTxGenerator` or something like this

Reply:
Renamed it to `tokensRemover`, since tools inside this file will not only generate txs